### ISSUE TITLE
No programs to find in PROGRAMMAS

### ISSUE THREAD

**Issue by cometoluc** [add-ons/plugin.video.vtm.go#337]
**Describe the bug**

**To Reproduce**
1.
2.
3.

**Expected behavior**

**Screenshots**

**Logs**
Add debug logs to help troubleshoot the issue. See https://kodi.wiki/view/Log_file/Easy for more info.

**System**
 - Addon version:
 - Kodi version:
 - Inputstream adaptive version:
 - Operating System (Windows / Mac OS / Android / LibreElec / OSMC / ...):
 - Special Hardware (RPI / Vero4K+ / ...):

**Additional context**

-----------------------------------------------------------------------------------------------------------------
Hallo,

hier ben ik weer met een probleem met de addon, als men in de categorie "programmas" gaat kijken is die leeg. Heb het vandaag pas opgemerkt. Als ik online naar vtmgo op pc ga daar heeft men een kategorie Home waar veel meer van de vtm programmas kan uitkiezen.

Kategorie films en my menu does work normally

Ik kan via kodi vb. the voice van vlaanderen niet uitkiezen, nieuws, het weer, alle programmas van spelprogrammas zoals regi, het jachtseizoen, voice kids, k3 zoekt k3, k2 zoekt k3 enz,
Opzoeken op de search vindt ze wel maar dan moet men de naam wel correct kennen.

Kan hier iemand naar kijken of me doorverwijzen waar ik het probleem kan melden in github aub als het niet hier zou zijn?

ik werk op windows 7.32bit laatste
vtmgo versie 1.3.2
matrix.1
kodi versie 19.3(19.3.0)

Sorry maar mijn engels is niet goed genoeg om dit in het engels te schrijven
Mvg
Luc Van devoorde
cometoluc@telenet.be

**Comment by michaelarnauts** [comment]
Al eens geprobeerd om in de instellingen van de add-on, bij expert naar "cache clearen" te gaan en dan opnieuw te proberen?

**Comment by cometoluc** [comment]
Ja, dat heb ik al geprobeerd. En heropgestart maar resultaat blijft hetzelfde.

Onder programmas staan gewoonlijk de programmas van vtm zelf.

Heb ik dan als enige dit probleem onder kodi op pc windows7.32

**Comment by cometoluc** [comment]
Hallo, is er niemand anders die dat voorheeft aub?

**Comment by davidbe** [comment]
Je bent niet de enige, ik heb zelfde ervaring: onder categorie films loopt alles blijkbaar zonder problemen, maar onder programma's inderdaad slechts beperkte titels. Er kan wel gezocht worden en dat wordt dan ook gevonden. Cache heb ik ook eens leeggemaakt.
Ik draai kodi op Rapberry Pi.

**Comment by cometoluc** [comment]
michaelarnauts

Sorry voor het storen maar hebt u al iets meer info of wat we kunnen proberen aub? Cache clearen helpt niet

**Comment by davidbe** [comment]
Ik probeer vanavond wat logging-informatie te verzamelen. @cometoluc , misschien kan je ook eens kijken?

**Comment by tatankat** [comment]
Volgens mij is de programma's toegang gewoon verwijderd uit vtm go. De toegang tot enigszins een lijst van programma's (en ook hoe de site en de app werken) gaat nu enkel via "Aanbevelingen".
Programma's zou dus verwijderd moeten worden, denk ik (en ik hoop dat er op een bepaald moment terug meer lokale cache bijgehouden zal worden, zodat de kodi app zelf een meer bruikbare lijst van programma's kan genereren).

**Comment by cometoluc** [comment]
ok mensen , dank u voor de opmerkingen, het zou spijtig zijn dat het niet terugkomt maar kan alleen vtm iets aan doen vrees ik, alleszins bedankt
